Makefile

```diff
@@ -210,7 +210,8 @@ GOLANGCI_LINT = $(LOCALBIN)/golangci-lint
 KUSTOMIZE_VERSION ?= v5.4.3
 CONTROLLER_TOOLS_VERSION ?= v0.16.1
 ENVTEST_VERSION ?= release-0.19
-GOLANGCI_LINT_VERSION ?= v1.62.2
+# v1.60.0 is the only version built using Go v1.23: https://github.com/golangci/golangci-lint/issues/4837
```

Comment:
This issue says that support started at 1.60, I would have expected that all versions above 1.60 to be compatible with Go 1.23

Reply:
@ahg-g I could only use v1.60.0 during initial testing. After further testing, this is no longer the case.